repl/transactions_commands.go

```diff
@@ -0,0 +1,167 @@
+package repl
+
+import (
+	"encoding/hex"
+	"fmt"
+	"strconv"
+
+	apitypes "github.com/spacemeshos/api/release/go/spacemesh/v1"
+	"github.com/spacemeshos/go-spacemesh/common/util"
+
+	"github.com/spacemeshos/CLIWallet/log"
+	gosmtypes "github.com/spacemeshos/go-spacemesh/common/types"
+)
+
+var transactionStateDisStringsMap = map[int32]string{
+	0: "Unspecified state",
```

Comment:
Is there a way we can move these strings into the API so that we can use harmonized strings everywhere?

Reply:
Maybe at a later stage if and when we support localized strings in the api. I don't think it is bad to have these on the client level right now.